carbon/assetlist.json

```diff
@@ -63,13 +75,25 @@
           {
             "denom": "bneo.1.14.e2e5f6",
             "exponent": 8,
-            "aliases": ["BNB"]
+            "aliases": ["bNEO"]
           }
         ],
         "base": "bneo.1.14.e2e5f6",
         "name": "BurgerNEO",
         "display": "bneo",
         "symbol": "bNEO",
+        "traces": [{
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "neo",
+            "base_denom": "bNEO",
```

Comment:
I don't know enough about NEO but this might supposed to be a contract address....